apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -3,7 +3,7 @@
 <span class="d-inline-block ms-2 pl-matrix-component-span-fullwidth">
     <div class="input-group">
         {{#label}}
-        <label class="input-group-text" id="basic-addon2">{{{label}}}</label>
+        <label class="input-group-text" id="pl-matrix-component-input-{{uuid}}-label">{{{label}}}</label>
```

Comment:
Is this really a useful label for _every single cell_ in the table? I'd expect cells to be labeled something like "Row 1, column 1".

In `pl-multiple-choice`, we use `<span role="group" aria-label="...">` to describe a group of inputs. Would that work here?

Reply:
Ah, I see that now. Sorry, I got confused by the two levels of labels.